WeatherNotFound/build.gradle.kts

```diff
@@ -48,6 +48,7 @@ dependencies {
     val appCompatVersion = "1.6.1"
 
     implementation("androidx.appcompat:appcompat:$appCompatVersion")
+    implementation("androidx.startup:startup-runtime:1.1.1")
```

Comment:
As you can see, in the Gradle file, the style of dependencies is like the following:

1. define the library version via a variable. like `appCompatVersion`
2. then use it in implementation section.


I will appreciate change this line and implement runtime library using two steps above.